data/db-schema/src/main/resources/net/corda/db/schema/config/migration/cpx-creation-v1.0.xml

```diff
@@ -195,7 +198,7 @@
             <column name="content" type="TEXT">
                 <constraints nullable="false"/>
             </column>
-            <column name="entity_version" type="INT">
```

Comment:
@benmcmahon100 - could you please put a description on this PR why this is necessary?
(I do not think Jira description would have any information on that)

Reply:
👍 Just working on that now